docs/disk-caching/DESIGN.md

```diff
@@ -49,5 +49,6 @@ Upon restart of minio server after a running minio process is killed or crashes,
 
 ## Limits
 - Bucket policies are not cached, so anonymous operations are not supported when backend is offline.
+- To ensure security guarantees, encrypted objects are never cached.
```

Comment:
We can add this as a TODO item, as we are planning to introduce encryption layer by taking a master key.

Reply:
done